doc/plugin_server_datastore_sql.md

```diff
@@ -135,6 +135,60 @@ If you need to use custom Root CA, just specify `root_ca_path` in the plugin con
     }
 ```
 
+### IAM Authentication
+
+With IAM database authentication, an authentication token is used instead of a password when connecting to the database. For that reason, a password must not be provided in the connection string.
```

Comment:
Thank you for the suggested change. Since IAM authentication is not tied to AWS IAM only (we will be supporting IAM authentication in other services from other cloud providers) the description should not be AWS-specific.
I'm taking this but without the AWS-specific parts. Thanks for the suggestion, it's a lot more clear! than the original text